src/main/java/bookstore/exception/SpecificationNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package bookstore.exception;
+
+public class SpecificationNotFoundException extends RuntimeException {
+    public SpecificationNotFoundException(String message) {
+        super(message);
+    }
+
+    public SpecificationNotFoundException(String message, Throwable cause) {
```

Comment:
I don't think you will use this constructor

Reply:
If you don't use some methods, you don't need to create them. Check the usage of this constructor please